### DISCUSSION TITLE
sanity/router error

### DISCUSSION THREAD

**Discussion by 3200pro** [sanity-io/sanity#3784] [Sanity Studio Support]
Hi Everyone:)

I'm getting an error with the majority of the V3 compatible plugins. They read:

`import {useIntentLink as $gGrEF$useIntentLink} from "sanity/router";
`

Any advice is appreciated:)

**Comment by kmelve** [comment]
What version of `sanity@dev-preview` are you on? And what does the _error_ actually say?

**Reply by 3200pro** [reply]
Hi @kmelve,

Thanks for the prompt assistance. I redid my migration and located the V3 plugin causing the problem:

[https://github.com/robinpyon/sanity-plugin-media/issues/97
](https://github.com/robinpyon/sanity-plugin-media/issues/97
)
